docs/user_guide.md

```diff
@@ -601,6 +600,418 @@ curl -s 'https://{CORGI_HOST}/api/v1/components?nvr=bare-metal-event-relay-opera
+"pkg:rpm/redhat/asio-devel@1.10.8-7.module%2Bel8%2B2765%2Bcfa4f87b?arch=ppc64le"
+"pkg:rpm/redhat/mariadb-debugsource@10.3.17-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/mariadb-gssapi-server@10.3.17-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/galera-debugsource@25.3.26-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/mariadb-oqgraph-engine@10.3.17-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/mariadb-errmsg@10.3.17-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/Judy@1.0.5-18.module%2Bel8%2B2765%2Bcfa4f87b?arch=i686"
+"pkg:rpm/redhat/mariadb-test@10.3.17-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/asio-devel@1.10.8-7.module%2Bel8%2B2765%2Bcfa4f87b?arch=x86_64"
+"pkg:rpm/redhat/mariadb-server@10.3.17-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/mariadb-oqgraph-engine-debuginfo@10.3.17-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/mariadb-server-galera@10.3.17-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/mariadb-embedded@10.3.17-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/mariadb-common@10.3.17-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/Judy-devel@1.0.5-18.module%2Bel8%2B2765%2Bcfa4f87b?arch=i686"
+"pkg:rpm/redhat/Judy-devel@1.0.5-18.module%2Bel8%2B2765%2Bcfa4f87b?arch=ppc64le"
+"pkg:rpm/redhat/asio-devel@1.10.8-7.module%2Bel8%2B2765%2Bcfa4f87b?arch=s390x"
+"pkg:rpm/redhat/asio-devel@1.10.8-7.module%2Bel8%2B2765%2Bcfa4f87b?arch=aarch64"
+"pkg:rpm/redhat/Judy-devel@1.0.5-18.module%2Bel8%2B2765%2Bcfa4f87b?arch=x86_64"
+"pkg:rpm/redhat/mariadb-devel@10.3.17-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/Judy-debuginfo@1.0.5-18.module%2Bel8%2B2765%2Bcfa4f87b?arch=i686"
+"pkg:rpm/redhat/mariadb-backup@10.3.17-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/mariadb-server-utils@10.3.17-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/mariadb@10.3.17-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/galera@25.3.26-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/mariadb-backup-debuginfo@10.3.17-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/mariadb-debuginfo@10.3.17-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/mariadb-server-debuginfo@10.3.17-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/mariadb-server-utils-debuginfo@10.3.17-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/mariadb-embedded-devel@10.3.17-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/mariadb-gssapi-server-debuginfo@10.3.17-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+"pkg:rpm/redhat/Judy-debugsource@1.0.5-18.module%2Bel8%2B2765%2Bcfa4f87b?arch=i686"
+"pkg:rpm/redhat/galera-debuginfo@25.3.26-1.module%2Bel8.1.0%2B3974%2B90eded84?arch=i686"
+```
```

Comment:
Doc updates LGTM, but the full output isn't needed I think. You can just give a few lines as a sample and then there's less scrolling involved for the user.